open_optical_gating/cli/cli.py

```diff
@@ -133,21 +159,21 @@ def initialise_internal_parameters(self):
         self.stop = False
 
     def init_hardware(self):
-        """Function that initialises various controlls (pins for triggering laser and fluorescence camera along with the USB for controlling the Newport stages)
+        """Function that initialises various hardware I/O interfaces (pins for triggering laser and fluorescence camera, along with the USB for controlling the Newport stages)
         Function inputs:
             laser_trigger_pin = the GPIO pin number connected to fire the laser
-            fluorescence_camera_pins = an array of 3 pins used to for the fluoresence camera
+            fluorescence_camera_pins = an array of 3 pins used to interact with the fluoresence camera
                                             (trigger, SYNC-A, SYNC-B)
             self.settings["usb_stages"] = a list containing the information used to set up the usb for controlling the Newport stages
                                     (USB address (str),timeout (flt), baud rate (int), byte size (int), parity (char), stop bits (int), xonxoff (bool))
-        Outputs:
-                usb_serial will be one of:
-                    0 - if no failure and no usb stage
-                    1 - if fastpins fails
-                    2 - if laser pin fails
-                    3 - if camera pins fail
-                    4 - if usb stages fail
-                    serial object - if no failure and usb stages desired
+                                    TODO: JT writes: no, this seems to be a dictionary, not a list...

```

Comment:
Yep, that's an update to make things less crazy that clearly hasn't been reflected in the doc string.